src/visualizers-workbench/P5VisualizerTemplate.ts

```diff
@@ -0,0 +1,178 @@
+import {P5Visualizer} from '../visualizers/P5Visualizer'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+
+/** md
```

Comment:
I don't see anywhere that the user is instructed to create markdown documentation for the visualizer.  Am I missing it?  Right here, you could put a comment above the markdown saying that every visualizer should include the following info in markdown, which will be scraped and compiled into the documentation page automatically...